documentation-website/cypress/e2e/license.cy.ts

```diff
@@ -11,7 +11,7 @@ describe('license', () => {
     cy.url().should('eq', BASE_URL + page.license,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'License | @idrinth/api-bench',);
+    cy.title().should('eq', 'License | @idrinth-api-bench/api-bench',);
```

Comment:
this package will never exist, please revert the change for now

Reply:
Reverted